front/app/components/admin/GraphCard/index.tsx

```diff
@@ -1,13 +1,13 @@
 import React from 'react';
 
 // components
-import { Box, Text, IconTooltip } from '@citizenlab/cl2-component-library';
 import {
-  GraphCard as GraphCardContainer,
-  GraphCardHeader,
-  GraphCardTitle,
-  GraphCardInnerClean,
-} from 'components/admin/GraphWrappers';
```

Comment:
Simplified this component, tried to get rid of as many `styled-components` as possible

Reply:
nice